### PR TITLE
[WIP] Adding pre-commit hooks configuration file

### PR DESCRIPTION
**Pull request by jakleh**

### CODE REVIEW

**Comment by jakleh** [comment]
> 

Forgot to mention in the pull-req. title: This is a checklist item for CAIRO integration.

**Comment by jakleh** [comment]
Also, I didn't mean to put WIP

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        language_version: python3.8
+
+  # General fixers: format files for white spaces and trailing new lines, warn on debug statements
+  # https://github.com/pre-commit/pre-commit-hooks#hooks-available
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: trailing-whitespace
+      - id: debug-statements
+      - id: end-of-file-fixer
```
Comment (taylorfturner) [inline .pre-commit-config.yaml:47]: @jakleh new line at EOF (end of file)

**Review by taylorfturner — COMMENTED** [review]
just one comment for now, but will review today. Thx for getting this up!

**Pull request closed without merging**